cmd/cinder-csi-plugin/main.go

```diff
@@ -94,6 +96,28 @@ func main() {
 }
 
 func handle() {
-	d := cinder.NewDriver(nodeID, endpoint, cluster, cloudconfig)
+	d := cinder.NewDriver(nodeID, endpoint, cluster)
+
+	//Intiliaze mount
+	mount, err := mount.GetMountProvider()
+	if err != nil {
+		klog.V(3).Infof("Failed to GetMountProvider: %v", err)
```

Comment:
If some error happens, does the csi plugin still work? Do we need to raise exception here?

Reply:
I thought about it, since previously it is already happening at later stage and we were continuing  with run and setup, at this point we can do the same.

Reply:
ok, keeping the behavior same with before is a good reason, probably we could improve that in a separate PR.

/lgtm